docs/examples/networking/hybrid_dns/env-vars

```diff
@@ -20,7 +20,7 @@ export TF_VAR_private_subnet_cidr=""
 
 ### Instance configuration
 export TF_VAR_InstanceShape=""
-export TF_VAR_InstanceImageDisplayName=""
+export TF_VAR_InstanceImageOCID=""
```

Comment:
What is the format if I were to provide this as an environment variable? Would this be a map?

Reply:
Good question... it is a bit cumbersome:

  export TF_VAR_InstanceImageOCID='{us-phoenix-1="ocid1.image.oc1.phx.aaaaaaaalfnxla4wjqnkehn44xzu74orzdo5pxtsytjfyry72h4n5zrfmwoq"}'